substrate/frame/transaction-storage/src/mock.rs

```diff
@@ -68,26 +71,20 @@ impl frame_system::Config for Test {
 	type MaxConsumers = ConstU32<16>;
 }
 
+#[derive_impl(pallet_balances::config_preludes::TestDefaultConfig as pallet_balances::DefaultConfig)]
```

Comment:
I would usually prefer unrelated changes to be in a separate PR

Reply:
neah, it's small change that can go here easily and avoid another PR